src/images/caret-down-white.svg

```diff
@@ -0,0 +1,14 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="-5 -5 26 26">
```

Comment:
Given this UI kit is public and universal, we should consider making these SVG's inlined and therefore also targetable by `fill` styles in CSS, rather than have two different color versions